app/src/connman.js

```diff
@@ -0,0 +1,87 @@
+var DBus, Promise, SERVICE, TECHNOLOGY_INTERFACE, WIFI_OBJECT, _, bus, config, dbus, fs, systemd, utils;
+
+Promise = require('bluebird');
+
+DBus = require('./dbus-promise');
+
+fs = Promise.promisifyAll(require('fs'));
+
+dbus = new DBus();
+
+bus = dbus.getBus('system');
+
+_ = require('lodash');
+
+config = require('./config');
+
+systemd = require('./systemd');
+
+utils = require('./utils');
+
+SERVICE = 'net.connman';
+
```

Comment:
All these linebreaks are hurting my eyes. We should pad them all with at least 3-4 blank lines.